pkgs/development/python-modules/boddle/default.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, bottle
+}:
+
+buildPythonPackage rec {
+  pname = "boddle";
+  version = "0.2.9";
+  disabled = pythonOlder "3.7";
```

Not required at the moment but perhaps in the future.

```suggestion
  version = "0.2.9";
  format = "setuptools";

  disabled = pythonOlder "3.7";
```